### PR TITLE
Fix formatting for proxy page

### PR DESCRIPTION
**Pull request by HeatherStenson**
These changes to the [proxy page](https://learning.postman.com/docs/getting-started/proxy/) were originally part of https://github.com/postmanlabs/postman-docs/pull/4065, but since that's on hold I wanted to move forward with them now!

### CODE REVIEW

File: src/pages/docs/getting-started/proxy.md [hunk 1]
```diff
@@ -76,19 +76,17 @@ Postman enables you to configure a custom proxy configuration that directs Postm
 
 [![Enable custom proxy](https://assets.postman.com/postman-docs/custom-proxy-flow.jpg)](https://assets.postman.com/postman-docs/custom-proxy-flow.jpg)
 
-Custom proxy settings are turned off by default. To turn them on, select **Settings** > **Add a custom proxy configuration**.
+Custom proxy settings are turned off by default. To turn them on, select **Settings** > **Add a custom proxy configuration** and update the following settings:
 
-Choose the type of proxy server by checking the appropriate checkboxes beside **Proxy Type**. By default, both `HTTP` and `HTTPS` are checked. This means that both HTTP and HTTPS requests will go through the proxy server.
+* **Proxy Type** - Choose the type of proxy server by selecting the appropriate checkboxes. By default, both `HTTP` and `HTTPS` are checked. This means that both HTTP and HTTPS requests will go through the proxy server.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/proxy.md:81]:
```suggestion
* **Proxy Type** - Choose the type of proxy server by selecting the appropriate checkboxes. By default, both `HTTP` and `HTTPS` are selected. This means that both HTTP and HTTPS requests will go through the proxy server.
```
```suggestion
* **Proxy Type** - Choose the type of proxy server by selecting the appropriate checkboxes. By default, both `HTTP` and `HTTPS` are checked. This means that both HTTP and HTTPS requests will go through the proxy server.
```

File: src/pages/docs/getting-started/proxy.md [hunk 2]
```diff
@@ -76,19 +76,17 @@ Postman enables you to configure a custom proxy configuration that directs Postm
...
+* **Proxy Server** - Enter the **host** or **IP address** (without protocol) and the **port** of the proxy server.
+* **Proxy Auth** - Postman uses the system's proxy configurations by default. To send API requests, select the option **This proxy requires authentication.** Turning this on makes the requests use Basic Authentication method, which requires a **Username** and a **Password**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/proxy.md:83]: "use Basic Authentication method" - should this be "use the Basic Authentication method"?

Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/proxy.md:83]: "use Basic Authentication method" - should this be "use the Basic Authentication method"?

**Review by mark-anthony-postman — APPROVED** [review]
Couple o' comments otherwise LGTM! 🌟

File: src/pages/docs/getting-started/proxy.md (hunk 2 above)
Comment (HeatherStenson) [inline src/pages/docs/getting-started/proxy.md:83]:
```suggestion
* **Proxy Auth** - Postman uses the system's proxy configurations by default. To send API requests, select the option **This proxy requires authentication.** Turning this on makes the requests use the Basic Authentication method, which requires a **Username** and a **Password**.
```

Comment (HeatherStenson) [reply src/pages/docs/getting-started/proxy.md]: Good catch, thank you!

**Pull request merged**